lib/Elastica/Index.php

```diff
@@ -295,7 +293,7 @@ public function exists()
      */
     public function createSearch($query = '', $options = null)
     {
-        $search = new Search($this->getClient());
+        $search = new Search($this->getClient(), $this->getClient()->getResultSetBuilder());
```

Comment:
The API might make more sense to have an optional ResultSetBuilder parameter in createSearch()? Personally I'd prefer to see not having an optional parameter and force it being set at creation time. It simplifies the code and any possibility of bugs (IME)